testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -380,22 +380,25 @@ def tearDown(self):
     def test_principal_axes(self):
         assert_array_almost_equal(
             self.ag.principal_axes(),
-            np.array([[-9.99925632e-01, 1.21546132e-02, 9.98264877e-04],
+            np.array([
+                      [1.53389276e-03, 4.41386224e-02, 9.99024239e-01],
                       [1.20986911e-02, 9.98951474e-01, -4.41539838e-02],
-                      [1.53389276e-03, 4.41386224e-02, 9.99024239e-01]]))
+                      [-9.99925632e-01, 1.21546132e-02, 9.98264877e-04]]))
 
     def test_align_principal_axes_with_self(self):
         pa = self.ag.principal_axes()
         self.ag.align_principal_axis(0, pa[0])
+
+
         assert_array_almost_equal(self.ag.principal_axes(), pa)
 
     def test_align_principal_axes_with_x(self):
-        self.ag.align_principal_axis(0, [1, 0, 0])
+        self.ag.align_principal_axis(0, [-1, 0, 0])
```

Comment:
The `-1` should not be necessary. I just tried
```python
In [9]: u.atoms.align_principal_axis(0, [1,0,0])
Out[9]: <AtomGroup with 3341 atoms>

In [10]: u.atoms.principal_axes()
Out[10]:
array([[  1.00000000e+00,   1.44968374e-08,  -7.78469679e-09],
       [ -1.49115813e-08,   9.98418272e-01,  -5.62223597e-02],
       [  6.95733711e-09,   5.62223597e-02,   9.98418272e-01]])
```
and that gives the identity to 0.06 or better.